src/main/java/seedu/letsgethired/ui/InternApplicationCard.java

```diff
@@ -14,6 +15,13 @@
 public class InternApplicationCard extends UiPart<Region> {
 
     private static final String FXML = "InternApplicationListCard.fxml";
+    private static final String ACCEPTED_COLOR = "-fx-text-fill: black; -fx-background-color: #7e38b7;";
+    private static final String ASSESSMENT_COLOR = "-fx-text-fill: black; -fx-background-color: #ffd100;";
+    private static final String INTERVIEW_COLOR = "-fx-text-fill: black; -fx-background-color: #fd5602;";
+    private static final String OFFERED_COLOR = "-fx-background-color: #03C04A;";
+    private static final String PANEL_COLOR = "-fx-background-color: #E9E9E9;";
+    private static final String PENDING_COLOR = "-fx-background-color: #209cee;";
+    private static final String REJECTED_COLOR = "-fx-background-color: #d30000;";
```

Comment:
Great choices of colours! I think it'll be better if these properties are set in the css file. This is not urgent.

Reply:
Thanks for the suggestion! I was only sure of how to conditionally set the colours in the Java file so i put it here. Same for the above odd/even issue